kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -50,6 +50,14 @@
         </div>
       </div>
 
+      <div
+        v-if="showTopicSizeWarning()"
+        class="shadow"
+        :style=" { padding: '1em', backgroundColor: $themePalette.grey.v_100 }"
```

Comment:
Just curious how this will look in the new LE's theme.

Reply:
Yeah that's a good point - @jtamiace mentioned to me that the colors palette will be reduced a bit so think like this might need to be updated when that happens.